### ISSUE TITLE
postgres may need md5 instead of sha

### ISSUE THREAD

**Issue by katestange** [numberscope/backscope#85]
In ubuntu install instructions, (this [line](https://github.com/numberscope/backscope/blob/210cb8544d976200bd864dc73cc7391f59fe7a72/doc/install-ubuntu.md?plain=1#L300)), on my system it was necessary to use md5 instead of scram-sha-256.  It just wouldn't authenticate with scram-sha-256.

Also, at this [line](https://github.com/numberscope/backscope/blob/210cb8544d976200bd864dc73cc7391f59fe7a72/doc/install-ubuntu.md?plain=1#L296), I didn't have a line matching that, I had `local all all md5`.  Maybe it is worth saying whether or not it matters what order these lines are in?  In case one doesn't have the exact line to put it in front of, as the instructions stipulate.

**Comment by katestange** [comment]
should I also just put this into #92?

**Comment by gwhitney** [comment]
Hmm, this seems more substantive, in that it seems like someone should start with a fresh Ubuntu and check out the differences you mention. Maybe it's a version difference? I.e., the instructions are for "Ubuntu latest" and you may have an earlier Ubuntu? Thinking of that, the install-ubuntu.md should definitely be edited to indicate exactly what version of Ubuntu it was last verified on, as "Ubuntu latest" is a moving target, but unless/until we edit them, our instructions are static. So that improvement could be folded into this issue as well. Anyhow, bottom line: this seems more contentful, whereas #92 seems more like exposition quality issues, so I would leave them separate.

**Comment by katestange** [comment]
Yes, I have an older ubuntu (20.04.6 LTS).  For some reason upgrading is failing and I haven't sat down to sort out why.
